deployment/aws-content-localization-auth.yaml

```diff
@@ -1,5 +1,5 @@
 AWSTemplateFormatVersion: "2010-09-09"
-Description: AWS Content Localization - Deploys the AWS Content Localization Application Cognito Infrastructure
+Description: "(SO0164) AWS Content Localization - Elasticsearch Consumer stack"
```

Comment:
Does tagging nested stacks cause us to count a single deployment multiple times?

Reply:
Not sure. I removed the solution id from nested stacks, just in case.

Reply:
Me either, the original CAS solution only had it in the top level stacks, so I was following that code.